packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -76,15 +77,33 @@ export class EpochTracker implements IPersistedFileCache {
     ): Promise<any> {
         try {
             const value: IVersionedValueWithEpoch = await this.cache.get(this.fileEntryFromEntry(entry));
+            // Version check
```

Comment:
nit: This comment doesn't add any new information. Same of the epoch check comment below.

Reply:
Would it be better to say that the epoch tracker is running 3 different checks on the get call, an epoch check, a version check and a cache expiry check or is that overkill?

Reply:
If you are adding a comment, it should say why we return undefined if the version does not match persistedCacheValueVersion - that's the part that is not obvious. Similarly, if the epoch does not match _fluidEpoch what does it mean?